pkg/infrastructure/services/dashboards.go

```diff
@@ -150,3 +156,8 @@ func deployGrafanaDashboards(dashboards []GrafanaDashboard, cli *client.Client,
 	}
 	return nil
 }
+
+func sanitizeDashboardName(name string) string {
```

Comment:
this func will remove any - or _ characters, is it expected to happen?

Reply:
Yes, but we can leave them. Fixing it